### ISSUE TITLE
Resolve the static nature of `Parser`

### ISSUE THREAD

**Issue by demberto** [demberto/PyFLP#4]
Possibly this can be implemented by making `Parser` a singleton and as a bonus add a `parse` method.

**Comment by demberto** [comment]
@ryanlong1004 What about static class counts? `_FLObject._count`? `_FLObject._ppq`?

**Comment by ryanlong1004** [comment]
I don't think those are necessary.

* Mostly, ```self``` is being used to call other class methods.
* You should never have to 'reset' static variables.  They should either be class methods to count instances or don't need to exist.
* All of the __private methods indicate (IMHO) they should be module methods that can, if needed, be called via an exposed class method.
* All of the self.__parse_xyz(ev), those should be props and ev should be a prop of that class.

**Comment by ryanlong1004** [comment]
>  """Reset static variables. This allows multiple instances of `Parser`
        in a single instance of the interpreter. HOWEVER THIS DOESN'T WORK FOR
        USING MULTIPLE INSTANCES AT THE SAME TIME."""

Use instances if you need to.

**Comment by demberto** [comment]
For a long amount of time I always thought of how I should structure the parser. I feel count is useless anyways. Your __parse_xyz idea is good but how do I implement it?

**Comment by ryanlong1004** [comment]
Can I do a little refactoring and submit a PR for you to look at?

**Comment by demberto** [comment]
Sure, sure 🤝

**Comment by demberto** [comment]
@ryanlong1004 any updates?

**Comment by ryanlong1004** [comment]
@demberto I did some work a week or so back.  Let me see if I can get a PR over.

**Comment by ryanlong1004** [comment]
I'm bowing out.  😎

**Comment by demberto** [comment]
Bruh.

**Comment by ryanlong1004** [comment]
Parser manipulates Events and the FLV(?) by directly changing each's *class* attributes to keep track.  Then there's a `reset` method that also acts directly on those class attributes as well.  

TL;DR it's too brittle and your app is too defendant on the Parser and I don't have enough time to be the guy that breaks it.

**Comment by demberto** [comment]
@ryanlong1004 It just resets the `_count` variable for all `_Event` and `_FLObject` subclasses. There is no `reset` method. Also what is "FLV"? Thanks anyways.

**Comment by demberto** [comment]
Fixed in [1.1.0](https://github.com/demberto/PyFLP/compare/1.0.1...1.1.0)
